src/pydata_sphinx_theme/assets/styles/extensions/_execution.scss

```diff
@@ -16,7 +16,7 @@
 }
 
 // Dark theme special-cases
-html[data-theme="dark"] .bd-content {
```

Comment:
Is this intentional? Just a quick thought that we probably don't need to rename our CSS prefix as part of this, right? (Unless we are just using the exact same section classes as basic-ng or something)

Reply:
Yes, because I am inheriting most of the HTML from sphinx-basic-ng, which has classes name 'sb-' etc. So had to update the CSS here. My thinking was that if we inherit the skeleton, we will just be updating the styles, etc inherited. Instead of updating the HTML as far as we can.